### PR TITLE
DIAL and DHCP Server: changed TRACE_LX to TRACE()

### PR DESCRIPTION
**Pull request by HaseenaSainul**

### CODE REVIEW

**Review by pwielders — APPROVED** [review]

File: DHCPServer/DHCPServer.cpp [hunk 1]
```diff
@@ -61,7 +61,7 @@ namespace Plugin {
 
         Core::Directory directory(_persistentPath.c_str());
         if (directory.CreatePath() == false) {
-            TRACE_L1("Could not create DHCPServer persistent path");
+            TRACE(Trace::Information, (_T("Could not create DHCPServer persistent path")));
```
Comment (pwielders) [inline DHCPServer/DHCPServer.cpp:64]: Error ?

File: DHCPServer/DHCPServer.cpp [hunk 2]
```diff
@@ -207,7 +207,7 @@ namespace Plugin {
                 leasesList.IElement::ToFile(leasesFile);
                 leasesFile.Close();
             } else {
-                TRACE_L1("Could not save leases in pemranent storage area.\n");
+                TRACE(Trace::Information, (_T("Could not save leases in permanent storage area.\n")));
```
Comment (pwielders) [inline DHCPServer/DHCPServer.cpp:210]: Error ?

**Review by pwielders — CHANGES_REQUESTED** [review]

File: DHCPServer/DHCPServer.cpp (hunk 1 above)
Comment (HaseenaSainul) [reply DHCPServer/DHCPServer.cpp]: done

File: DHCPServer/DHCPServer.cpp (hunk 2 above)
Comment (HaseenaSainul) [reply DHCPServer/DHCPServer.cpp]: done

**Review by pwielders — APPROVED** [review]

**Pull request merged**